spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET /posts' do
+    before(:example) { get '/users/:id/posts' }
+    it 'should check the responce is comming' do
+      expect(response).to have_http_status(:ok)
+    end
+    it('is correct template rendered') do
+      expect(response).to render_template(:index)
+    end
+  end
+  describe 'GET /posts/:id' do
+    # before(:example) { get '/users/:users_id/posts/1' }
+    before do
```

Comment:
- [OPTIONAL] Kindly remove the code that is commented. Removing such codes makes your work look clean and tidy. 👍 😎

Reply:
@Gambit142  Thank you for your review I make sure to update the my code